File: monitorlibrary/lib/data/answer.dart
import 'package:meta/meta.dart';

import 'content.dart';

class Answer {
  String? text, aNumber, created;
  List<Content> photoUrls = [], videoUrls = [];

  Answer(
      {required this.text,
      required this.aNumber,
      required this.photoUrls,
      required this.videoUrls});

  Answer.fromJson(Map data) {
    this.text = data['text'];
    this.aNumber = data['aNumber'];
    this.photoUrls = data['photoUrls'];
    this.videoUrls = data['videoUrls'];
    this.created = data['created'];
  }
  Map<String, dynamic> toJson() {
    List mPhotos = [];

    photoUrls.forEach((photo) {
      mPhotos.add(photo.toJson());
    });

    List mVideos = [];

    videoUrls.forEach((photo) {
      mVideos.add(photo.toJson());
    });

    Map<String, dynamic> map = {
      'text': text,
      'aNumber': aNumber,
      'created': created,
      'photoUrls': mPhotos,
      'videoUrls': mVideos,
      'created': created,
    };
    return map;
  }
}

File: webapi-spring/src/main/java/com/monitorz/webapi/data/Answer.java
package com.monitorz.webapi.data;

import java.util.List;

public class Answer {
    private String text, aNumber, created;
    private List<Content> photoUrls, videoUrls;
    private Respondent respondent;

    public Respondent getRespondent() {
        return respondent;
    }

    public void setRespondent(Respondent respondent) {
        this.respondent = respondent;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getaNumber() {
        return aNumber;
    }

    public void setaNumber(String aNumber) {
        this.aNumber = aNumber;
    }

    public String getCreated() {
        return created;
    }

    public void setCreated(String created) {
        this.created = created;
    }

    public List<Content> getPhotoUrls() {
        return photoUrls;
    }

    public void setPhotoUrls(List<Content> photoUrls) {
        this.photoUrls = photoUrls;
    }

    public List<Content> getVideoUrls() {
        return videoUrls;
    }

    public void setVideoUrls(List<Content> videoUrls) {
        this.videoUrls = videoUrls;
    }
}
